e2e/nomostest/testutils/fleet_membership.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
+
+import (
+	"fmt"
+	"strings"
+
+	"kpt.dev/configsync/e2e/nomostest"
+	"kpt.dev/configsync/pkg/reconcilermanager"
+)
+
+// TestCrossProjectFleetProjectID is the fleet host project.
+const TestCrossProjectFleetProjectID = "cs-dev-hub"
+
+// RegisterCluster registers a cluster in a fleet.
+func RegisterCluster(nt *nomostest.NT, fleetMembership, fleetProject, gkeURI string) error {
```

Comment:
Do these functions need to be public? Looks like they are only intended to be used within this package

Reply:
Changed it back to package scoped